apps/end-tenancy/controllers/loop.js

```diff
@@ -34,20 +34,27 @@ module.exports = class LoopController extends DateController {
 
   get(req, res, callback) {
     if (!req.params.action ||
-      Object.keys(this.options.subSteps).indexOf(req.params.action) === -1 ||
-      !req.params.edit &&
-      _.some(this.options.subSteps[req.params.action].prereqs, prereq =>
-        req.sessionModel.get(prereq) === undefined
-      )
+      Object.keys(this.options.subSteps).indexOf(req.params.action) === -1
```

Comment:
Sure, I'll update, this loop controller is being generalised and abstracted just as soon as I get time though